src/cpp/lib/QtWidgets/QMenuBar/qmenubar_wrap.cpp

```diff
@@ -15,6 +16,8 @@ Napi::Object QMenuBarWrap::init(Napi::Env env, Napi::Object exports) {
   Napi::Function func = DefineClass(
       env, CLASSNAME,
       {InstanceMethod("addMenu", &QMenuBarWrap::addMenu),
+       InstanceMethod("addMenuWithName", &QMenuBarWrap::addMenuWithName),
```

Comment:
Similar reason as https://github.com/nodegui/nodegui/pull/347/files#r365557394
I dont think we should have addMenuWithName

Reply:
or if we do then we do this on JS side only. 
where addMenu(string) would essentially create a new QMenu() on js side and then pass it to c++.

Reply:
Alright, I will just add it on it on just the JS side at least to save some of the verbosity of creating a menu.